src/esp/gfx/Viewer.cpp

```diff
@@ -87,8 +90,17 @@ Viewer::Viewer(const Arguments& arguments)
   agentBodyNode_ = &rootNode_->createChild();
   cameraNode_ = &agentBodyNode_->createChild();
 
+  /*
   cameraNode_->translate({0.0f, cameraHeight, 0.0f});
   agentBodyNode_->translate({0.0f, 0.0f, 5.0f});
+  */
+
+  vec3f eye(0.0f, 0.0f, 1.0f);
```

Comment:
This is the debug code, and it is temporary. Will definitely remove them after it is finished.